packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -279,6 +281,45 @@ void main() {
         expect(timeline.events.single.name, 'test event');
       });
     });
+
+    group('sendCommand error conditions', () {
+      test('local timeout', () async {
+        when(mockIsolate.invokeExtension(any, any)).thenAnswer((Invocation i) {
+          return new Future<Map<String, dynamic>>.delayed(
```

Comment:
Presumably you want this function to never return a value. Therefore, can this be simply:

```dart
return new Completer<Map<String, dynamic>>.future;
```

?